### DISCUSSION TITLE
사이드바 열림/닫힘 적용 상태로, 사이드바를 활성화시 블러처리된 부분 리스트 클릭시 페이지 진입 현상

### DISCUSSION THREAD

**Discussion by MUKER-WON** [pronist/hELLO#255] [Ideas]
<img width="1857" alt="스크린샷 2024-09-24 오전 9 38 33" src="https://github.com/user-attachments/assets/10a3d7d8-c84a-4dd3-9d4a-2d16530fcc86">

먼저 멋진 스킨을 공유해주셔서 잘 사용하고 있습니다. 감사합니다!!!!!!!!!

스크린샷과 같이 사이드바가 활성화 되어있을 시 블러처리된 리스트(빨간 스퀘어)를 클릭하면 네비게이션이 되는게 어색해서 건의드립니다!
블러처리 된 상태에서는 어디를 누르든 네비게이션이 활성화 되지 않고 사이드바가 들어가는 상황은 어떨까요..?

**Comment by pronist** [comment]
@MUKER-WON 안녕하세요.

그것은 버그입니다. 이전 버전에는 안 그랬었다가 커버 관련 코드를 건드리면서 생긴 버그입니다.
다음 패치 때 처리할 예정이에요 😊

**Reply by MUKER-WON** [reply]
다음 버전이 기대가 되네요! 감사합니다 좋은 하루 보내세요~
